hello-world/2025_03_08_international_womens_day.md

```diff
@@ -0,0 +1,48 @@
+# A tribute to two women who enabled my career
+
+## My great-grandmother
+My great-grandmother (the mother of the mother of my mother) is without any
+doubt someone who inspired my career, even if she did not live to see the result.
+
+Born in april 1895, she was the only relative I ever knew who was born in the
+19th century. Se was the youngest of thirteen children wo became a kindergarden
+teacher until she married (she was fired as result, which was common law back
+in the day in the Netherlands, and not only there). 
+
+Her housband died when she was just 32 years old, mother of 3 daughters, in the
+age of 5 (my greandmother), 3, and 1 year old. She rebranded the store she and
+her husband husband owned, from furnature to yarn (as no men in these days would
+buy furnature from a woman, or take her advise on the subject).
+
+She sent all her daughters to the [MULO](https://en.wikipedia.org/wiki/Meer_Uitgebreid_Lager_Onderwijs),
+a form of extended primary education which was definitaly frowned up by some,
+as it was not considered useful for women to get educated beyond the bare
+minimum in the 30's of the previous centery. She did it anyway because she
+strongly believed that education was the only way to become financially
+independent.
+
+That mindset shaped both my grandmother and mother and where key to how also
+I was raised, and now raise my daughter.  The fact that she lived another
```

Comment:
she => my grandmother.
But that's just my preference, after reading entire sentence it becomes clear that "she" is not about your daughter ;).